src/core/api.js

```diff
@@ -5,10 +5,16 @@ const ApiService = {
     axios.defaults.baseURL = "http://react-cdp-api.herokuapp.com/";
   },
   getMovies(params) {
-    return axios.get("movies", { params: params }).then(result => result.data);
+    return axios.get("movies", { params: params }).then(response => {
```

Comment:
This ApiService object is not loosely coupled with the implementation details (axios could not be easily swapped with a different ajax service).
Consider a more general approach.